Koloda.podspec

```diff
@@ -2,7 +2,7 @@
 
 Pod::Spec.new do |s|
 s.name             = "Koloda"
-s.version          = "3.1.2"
+s.version          = "3.1.3"
```

Comment:
You shouldn't have updated the version because for swift 3 release will have new major version